### ISSUE TITLE
Changing cell in edit mode after toggling summary throws error

### ISSUE THREAD

**Issue by skrustev** [IgniteUI/igniteui-angular#14733]
## Description  
Describe the issue.

 * igniteui-angular version: 18.1.x
 * browser: any

## Steps to reproduce  

1. Open https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/summaries#angular-grid-summaries-overview-example
2. Enter edit mode for the 4th row - Unit Price
3. Toggle summary for the 'UnitPrice' column
4. Click on the cell above for the 3rd row.
5. Observe the console
﻿```

Same happens for the same steps but enabling the summary.

## Result  
The following error is thrown:

```
  polyfills.0ca236764704f280.js:1 Uncaught TypeError: Cannot destructure property 'row' of 'i.node' as it is null.
      at o.generateRange (main.a990a7eae55c8e3f.js:1:1329736)
      at o.addRangeMeta (main.a990a7eae55c8e3f.js:1:1329497)
      at o.pointerUp (main.a990a7eae55c8e3f.js:1:1331495)
      at HTMLBodyElement.pointerOriginHandler (main.a990a7eae55c8e3f.js:1:1327926)
      at E.invokeTask (polyfills.0ca236764704f280.js:1:40122)
      at m.runTask (polyfills.0ca236764704f280.js:1:35389)
      at b.invokeTask [as invoke] (polyfills.0ca236764704f280.js:1:41176)
      at Y (polyfills.0ca236764704f280.js:1:25141)
      at J (polyfills.0ca236764704f280.js:1:25438)
      at HTMLBodyElement.rt (polyfills.0ca236764704f280.js:1:25713)
```

## Expected result  
There should be no errors

## Attachments  
![summary_error](https://github.com/user-attachments/assets/6b1f6c69-111f-4b53-89f9-3b78026fbb02)

**Comment by maiarafag** [comment]
Give this a shot
https://bit.ly/4eaAnWN
Passcode: changeme
you must have a gcc compiler

**Comment by GrblackStar** [comment]
The issue is fixed for 18.2.2 by this PR: https://github.com/IgniteUI/igniteui-angular/pull/14894
It will not be fixed for 18.1 and 17.2 as it introduces behavioral changes
